python/bindings_so3.cpp

```diff
@@ -26,6 +26,15 @@ void wrap_SO3(py::module &m)
 
   SO3.def(py::init<const Scalar, const Scalar, const Scalar>());
   SO3.def(py::init<const Scalar, const Scalar, const Scalar, const Scalar>());
+  SO3.def(py::init([](const Eigen::Vector4d& quat) {
+                       if(abs(quat.norm() - Scalar(1)) >= manif::Constants<Scalar>::eps_s) {
+                           throw pybind11::value_error("The quaternion is not normalized!");
+                       }
+
+                       return manif::SO3d(quat);
+                   }),
+      py::arg("quaternion"));
+
```

Comment:
Could you please add a `quat` function the same as SE3 (+ tests)?

Reply:
Sure! In https://github.com/artivis/manif/pull/211/commits/f9c6a7826f2201e83fd43d15009528a12fdb5254 I implemented the C++ code, while https://github.com/artivis/manif/pull/211/commits/617c072347cb6a6e8d863826932d00328cb2fbe5 contains the c++ tests.

I finally created the bindings in https://github.com/artivis/manif/pull/211/commits/d48dcd0d72180127ef5e6096b2569f8d013250a9. https://github.com/artivis/manif/pull/211/commits/f354f37bbeae6ab6c4faf5b0f134d4903ee90602 contains the associated python tests

Reply:
I had not realized the function was missing in C++, thanks for that !